lib/house.rb

```diff
@@ -0,0 +1,49 @@
+class House
+  attr_reader :price, :address, :rooms, :details
+
+  def initialize(price, address)
+    @price = price[1..-1].to_i
+    @address = address
+    @rooms = []
+    @total_area = 0
+    @details = {"price" => @price,
+                "address" => @address
+               }
+  end
+
+  def add_room(room_object)
+    @rooms << room_object
+  end
+
+  def above_market_average?
+    if @price > 500000 then true else false end
```

Comment:
as a matter of convention, you would not see a single line if/else/end like this.  If you're going to do it single line, you'd see it as a ternary. 

Technically, you don't need the explicit true/false. I'd do this:

```
def above_market_average?
   @price > 500000
end
```

Since that statement is a comparison, it returns the correct boolean itself, without needing an explicit true/false return.

Reply:
Ruby ternary, if you're interested: https://www.rubyguides.com/2019/10/ruby-ternary-operator/